Modules/Utils.cs

```diff
@@ -621,60 +627,91 @@ public static void NotifyRoles(bool isMeeting = false, PlayerControl SpecifySeer
                         if (target == seer) continue;
                         TownOfHost.Logger.Info("NotifyRoles-Loop2-" + target.GetNameWithRole() + ":START", "NotifyRoles");
 
+                        bool showToGhosts = seer.Data.IsDead && Options.GhostCanSeeOtherRoles.GetBool();
+
                         //他人のタスクはtargetがタスクを持っているかつ、seerが死んでいる場合のみ表示されます。それ以外の場合は空になります。
-                        string TargetTaskText = seer.Data.IsDead && Options.GhostCanSeeOtherRoles.GetBool() ? $"{GetProgressText(target)}" : "";
+                        string TargetTaskText = showToGhosts ? $"{GetProgressText(target)}" : "";
+
+                        //他人の役職とタスクは幽霊が他人の役職を見れるようになっていてかつ、seerが死んでいる場合のみ表示されます。それ以外の場合は空になります。
+                        string TargetRoleText = showToGhosts ? $"<size={fontSize}>{Helpers.ColorString(target.GetRoleColor(), target.GetRoleName())}{TargetTaskText}</size>\r\n" : "";
+
+                        //RealNameを取得 なければ現在の名前をRealNamesに書き込む
+                        string TargetPlayerName = target.GetRealName(isMeeting);
+
+                        string TargetDeathReason = "";
 
                         //名前の後ろに付けるマーカー
                         string TargetMark = "";
+
                         //呪われている人
                         if (Main.SpelledPlayer.Find(x => x.PlayerId == target.PlayerId) != null && isMeeting)
                             TargetMark += "<color=#ff0000>†</color>";
-                        //タスク完了直前のSnitchにマークを表示
-                        canFindSnitchRole = seer.GetCustomRole().IsImpostor() || //Seerがインポスター
-                            (Options.SnitchCanFindNeutralKiller.GetBool() && seer.Is(CustomRoles.Egoist));//or エゴイスト
 
-                        if (target.Is(CustomRoles.Snitch) && canFindSnitchRole)
+                        if (seer.Data.IsDead)
                         {
-                            var taskState = target.GetPlayerTaskState();
-                            if (taskState.DoExpose)
-                                TargetMark += $"<color={GetRoleColorCode(CustomRoles.Snitch)}>★</color>";
-                        }
+                            //霊界からラバーズ視認
+                            if (target.Is(CustomRoles.Lovers))
+                            {
+                                TargetMark += $"<color={GetRoleColorCode(CustomRoles.Lovers)}>♡</color>";
+                            }
 
-                        //ハートマークを付ける(相手に)
-                        if (seer.Is(CustomRoles.Lovers) && target.Is(CustomRoles.Lovers))
-                        {
-                            TargetMark += $"<color={GetRoleColorCode(CustomRoles.Lovers)}>♡</color>";
+                            foreach (var ExecutionerTarget in Main.ExecutionerTarget)
+                            {
+                                if (target.PlayerId == ExecutionerTarget.Value)
```

Comment:
Targets of other Executioners are visible

Reply:
The dead can see all targets, should only dead Executioners see their failed target?